pkg/scheduler/factory.go

```diff
@@ -93,6 +94,9 @@ type Config struct {
 	// Recorder is the EventRecorder to use
 	Recorder events.EventRecorder
```

Comment:
this is already an interface... why would we not implement this interface using the v1 Events API?